sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -127,6 +140,7 @@ def get_latest_feature_values_from_dataframes(driver_df, customer_df, entity_row
     latest_customer_row = customer_rows.loc[
         customer_rows["event_timestamp"].idxmax()
     ].to_dict()
+    latest_global_row = global_df.loc[global_df["event_timestamp"].idxmax()].to_dict()
 
     latest_customer_row.update(latest_driver_row)
-    return latest_customer_row
+    return {**latest_customer_row, **latest_driver_row, **latest_global_row}
```

Comment:
If you're merging the dictionaries like this, you can remove the line above.

Reply:
my mistake, forgot to remove that line